pkg/vm/vm.go

```diff
@@ -232,7 +232,7 @@ func (v *VM) Load(prog []byte) {
 	v.LoadScript(prog)
 }
 
-// LoadScript loads a script from the internal script table. It
+// LoadScript loads a script from the test_util script table. It
```

Comment:
It looks strange... Automatic refactoring? :D

Reply:
it is, it is. Without it it will be hard to make refactorings =)